runbot/runbot.py

```diff
@@ -1263,7 +1263,7 @@ def branch_info(branch):
             })
 
         # consider host gone if no build in last 100
-        build_threshold = max(build_ids) - 100
+        build_threshold = max(build_ids or [0]) - 100
```

Comment:
This patch fix it... no ?
 
-> max(None) -> crash: TypeError: 'NoneType' object is not iterable
=> max(None or [0]) -> 0

->  max([]) -> crash: ValueError: max() arg is an empty sequence
=>  max([] or [0]) -> 0